tests/auth_tests/test_forms.py

```diff
@@ -867,6 +869,31 @@ def test_bug_19349_bound_password_field(self):
         # value to render correctly
         self.assertEqual(form.initial["password"], form["password"].value())
 
+    @override_settings(ROOT_URLCONF="auth_tests.urls_admin")
+    def test_bug_34066_link_to_password_reset_in_helptext(self):
+        user = User.objects.get(username="testclient")
+        form = UserChangeForm(data={}, instance=user)
+        passsword_help_text = form.fields["password"].help_text
+        matches = re.search('<a href="(.*?)">', passsword_help_text)
+
+        # Get URL to UserChangeForm in admin via to_field (instead of pk)
+        admin_user_change_url = f"""{reverse(
```

Comment:
Django coding standards prevent non-trivial f string interpolation I'm afraid 🙂 … (although tests might be another story and Carlton may be ok with it)

Reply:
Yes, f-strings should use only plain variable and property access as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).